Update README.md to include mysql-client install (#246)
planetscale/cli: README.md

```
brew install planetscale/tap/pscale
```

> !! Note of importance: `pscale` requires the MySQL Client. You can install it by running:

Reviewer: We only require it for the `pscale shell` subcommand, otherwise, it's optional. I wonder if should add this bit in paranthese? 

```suggestion
Optional: `pscale` requires the MySQL Client for certain commands. You can install it by running:
```

Author: As a user, it would make perfect sense to me so I'm keeping my eyes peeled to see what you decide.

Reviewer: Thank you Edgar.

Author: Your welcome Fatih, its an awesome db platform😄
